recipe/meta.yaml

```diff
@@ -26,32 +24,37 @@ build:
 
 requirements:
   host:
-    - debugpy >=1.0.0,<2.0
+    - python
+    - debugpy >=1.0.0
+    - hatchling >=1.4
     - ipython >=7.23.1
-    - jupyter_client <8.0
-    - jupyter_core >=4.2
+    - jupyter_client >=6
+    - packaging
     - pip
-    - python
-    - setuptools
+    - psutil
     - wheel
   run:
+    - python
     - appnope  # [osx]
-    - debugpy >=1.0.0,<2.0
+    - debugpy >=1.0.0
     - ipython >=7.23.1
-    - jupyter_client <8.0
-    - matplotlib-inline >=0.1.0,<0.2.0
+    - jupyter_client >=6.1.12
+    - matplotlib-inline >=0.1.0
     - nest-asyncio
-    - python
-    - tornado >=4.2,<7.0
-    - traitlets >=5.1.0,<6.0
+    - packaging
+    - psutil
+    - pyzmq >=17
+    - tornado >=6.1
```

Comment:
is it save to give up upper bounds here?

Reply:
I've added the upper bound again for `tornado` and `traitlets`